Tools/autotest/common.py

```diff
@@ -5780,7 +5780,7 @@ def validator(value2, target2=None):
             target=altitude_min,
             current_value_getter=lambda: self.get_altitude(
                 relative=relative,
-                timeout=timeout,
+                timeout=0.05,
```

Comment:
that seems wrong as you disable the maintain state and will mostly always trigger a timeout exception